scss/_variables.scss

```diff
@@ -1037,7 +1041,7 @@ $thumbnail-box-shadow:              $box-shadow-sm !default;
 
 // Figures
 
-$figure-caption-font-size:          90% !default;
+$figure-caption-font-size:          $small-font-size !default;
```

Comment:
Isn't this smaller than what it was?

Reply:
It is, but the change it minimal (from `14.4px` to `14px`). As I said, I tweaked some font sizes to make things more consistent.